setup.cfg

```diff
@@ -27,7 +27,7 @@ project_urls =
 [options]
 package = find:
 include_package_data = True
-python_requires = >3.9
+python_requires = >=3.10
```

Comment:
I like personally new Python versions, but for a scientific library, the 3 minor versions we support now was already on the tighter end. Actually, NumPy just dropped 3.8 [today](https://numpy.org/neps/nep-0029-deprecation_policy.html) so we are only starting to be NEP29-compliant.

Reply:
Thanks for being this quick.

So this might be a bug because it says > (**greater**) than 3.9.
I'm happy to change to >= (**greater or equal**) than 3.9, it's just that pre-commit was not working with the **greater** symbol 😂 so I had to switch to >= 3.10.

Reply:
Bug indeed... Never noticed since local and CI tests has been running fine on 3.9 and the line confusingly has a `=` right before the `>`.

Reply:
Actually I will open a separate PR for this

Reply:
See #110.

Reply:
Sure, I merged with main.